pkg/ddevapp/config.go

```diff
@@ -1524,18 +1515,6 @@ func PrepDdevDirectory(app *DdevApp) error {
 		}
 	}
 
-	// Pre-create a few dirs so we can be sure they are owned by the user and not root.
-	dirs := []string{
-		"web-entrypoint.d",
-		"xhprof",
-	}
-	for _, subdir := range dirs {
-		err = os.MkdirAll(filepath.Join(dir, subdir), 0755)
-		if err != nil {
-			return err
-		}
-	}
-
```

Comment:
I remember we had problems with this before, and tests failed here (again), when there was no `.ddev/web-build` after `ddev config`.

I decided to add `PopulateExamplesCommandsHomeadditions` on `ddev config` (see above), and added `web-entrypoint.d/README.txt`, so that this directory would be created as well.

Reply:
Alternatively, I could add `web-build` and `db-build` here to the `dirs` slice - but why would I do that if we can add all the assets to the `ddev config`.